examples/gke_cluster/variables.tf

```diff
@@ -3,3 +3,59 @@ variable "castai_api_token" {}
 variable "project_id" {}
 variable "cluster_region" {}
 variable "cluster_name" {}
+
+variable "castai_api_url" {
```

Comment:
I think this should be removed, by default we use it and only adds confusion for external users